varya/assets/sass/structure/_responsive-logic.scss

```diff
@@ -181,9 +181,9 @@ $breakpoint_xxl: 1024px;
 @include media(mobile) {
 	%responsive-alignleft {
 		/*rtl:ignore*/
-		margin-left: auto;
+		margin-left: var(--responsive--alignleft-margin);
```

Comment:
I'm encountering a small bug when I use the `alignleft` class:

Left aligned cover block | No alignment cover block
-------- | --------
<img width="688" alt="Screen Shot 2020-04-24 at 12 08 33 PM" src="https://user-images.githubusercontent.com/5375500/80233796-10ae6d00-8625-11ea-83d7-e1bb6062d63e.png">|<img width="665" alt="Screen Shot 2020-04-24 at 12 08 54 PM" src="https://user-images.githubusercontent.com/5375500/80233879-32a7ef80-8625-11ea-838e-f5f6f937ebd6.png">


The left aligned cover block is slightly out of alignment with the default content width. Should this be a separate issue that tackles the way the `--responsive--alignleft-margin` value is calculated?

Reply:
Hmmm. Having trouble replicating this one. Could double check that this is still happening for you with the latest version of Gutenberg?

Reply:
This is still happening, but it's not a result of this PR.